checkov/common/bridgecrew/vulnerability_scanning/image_scanner.py

```diff
@@ -109,5 +113,28 @@ def scan(self, docker_image_id: str, dockerfile_path: str, skip_extract_image_na
             logging.error("Failed to scan docker image", exc_info=True)
             return 1
 
+    def get_scan_results_from_cache(self, image_id: str) -> Dict[str, Any]:
+        image_id_encode = quote_plus(':'.join(['sha256', image_id]))
```

Comment:
```suggestion
        image_id_encode = quote_plus(f"sha256:{image_id}")
```
better to use a f-string literal here, also looks nicer 😉